lib/vtls/sectransp.c

```diff
@@ -1453,18 +1265,57 @@ static bool is_cipher_suite_strong(SSLCipherSuite suite_num)
   return true;
 }
 
-static bool sectransp_is_separator(char c)
+static int sectransp_cipher_suite_get_str(uint16_t id, char *buf,
```

Comment:
These wrapper functions for `Curl_cipher_suite_get_str` and `Curl_cipher_suite_walk_str` are here so there is no change in behavior with this commit. But, I guess no one will shed a tear if support for setting/getting these ciphers is removed?